lib/ast.js

```diff
@@ -371,9 +373,9 @@ var AST_Lambda = DEFNODE("Lambda", "name argnames uses_arguments", {
     _walk: function(visitor) {
         return visitor._visit(this, function(){
             if (this.name) this.name._walk(visitor);
-            this.argnames.forEach(function(arg){
-                arg._walk(visitor);
-            });
+            for (var i = 0, len = this.argnames.length; i < len; i++) {
+                this.argnames[i]._walk(visitor);
```

Comment:
I don't think this kind of micro-optimization will save anything these days.  It only makes code uglier.